Automattic/wp-calypso
# Domains: Update free WordPress address component

#### Changes proposed in this Pull Request

This PR depends on #57111 for the white background CSS class. This is part of the domain managements page redesign project, detailed in pcYYhz-m2-p2.

This PR redesigns the free WPCOM address component which is located at the bottom of the domain management list page. It's main updates were the addition of a background and replacing the popover menu button with an ellipsis menu.

Desktop view screenshots, with and without the primary site address badge:

<img width="840" alt="Screen Shot 2021-10-18 at 23 57 38" src="https://user-images.githubusercontent.com/5324818/137836946-310f4bbb-8152-4791-b14b-da73a6e298a0.png">
<img width="844" alt="Screen Shot 2021-10-18 at 23 54 53" src="https://user-images.githubusercontent.com/5324818/137836941-ba585751-8c55-438a-99d0-497626f7d5b2.png">

Mobile view screenshots, with and without the primary site address badge:

![Annotation on 2021-10-18 at 23-58-14](https://user-images.githubusercontent.com/5324818/137837116-32419c00-6c0a-4567-beea-b4290094add4.png)
![Annotation on 2021-10-19 at 00-06-24](https://user-images.githubusercontent.com/5324818/137837118-79750355-0e98-48e5-ace1-d59cb95d4867.png)

#### Testing instructions

Access the live Calypso link, go to "Upgrades > domains" and append this feature flag to your browser's URL: `?flags=domains/management-list-redesign`. Ensure the page contains the free WordPress address component and it looks like the screenshots above.
